tests/plan/import/local-and-remote/main.fmf

```diff
@@ -3,5 +3,5 @@ execute:
 
 plan:
   import:
-    url: https://github.com/teemtee/tmt
-    name: /plans/sanity/lint
+    url: https://github.com/teemtee/tests
+    name: /plans
```

Comment:
```suggestion
    name: /plans/remote
```

I guess, this should point to a single plan as before?

Reply:
Yep, read the test wrong on that one.